src/esphome/sensor/cse7766.cpp

```diff
@@ -27,7 +27,6 @@ void CSE7766Component::loop() {
     if (!this->check_byte_()) {
       this->raw_data_index_ = 0;
       this->status_set_warning();
-      continue;
```

Comment:
Why was this continue removed? Without this, the code will accept any packet even if the checksum is invalid. We should not allow that.